x/mongo/driver/topology/connection.go

```diff
@@ -30,6 +30,9 @@ import (
 
 var globalConnectionID uint64 = 1
 
+var defaultMaxMessageSize uint32 = 48000000
```

Comment:
nit: use a single `var` block:

```
var (
    defaultMaxMessageSize uint32 = ...
    errResponseTooLarge = errors.New(...)
```

Note that you don't have to declare that the type of `errResponseTooLarge` is `error`. It's inferred from the declaration.

Reply:
Ah nice, forgot about `var` blocks 💯